resources/function_help/json/array_filter

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "array_filter",
+  "type": "function",
+  "description": "Returns an array with only the items for which the expression evaluates to true.",
+  "arguments": [
+    {"arg":"array","description":"an array"},
+    {"arg":"expression","description":"an expression to evaluate on each item. The variable `@element` will be replaced by the current value."}
+  ],
+  "examples": [
+    { "expression": "array_filter(array(1,2,3),@element < 3)", "returns":"array: [1, 2]"}
```

Comment:
It's the first help adding square brackets in the example. Other files are usually a copy/paste from the expression output preview. Why not doing the same?

Reply:
Good point.
Taking a step back, wouldn't it make sense to adjust the rest (without proposing this for this pull request)? `[1, 2, 3]` looks much more array like to me.